north/src/main.rs

```diff
@@ -19,10 +19,14 @@
 extern crate structure;
 
 use anyhow::{Context, Error, Result};
+use async_std::path::PathBuf;
+use async_std::prelude::*;
```

Comment:
should be formated like `use async_std::{fs, path::PathBuf, prelude::*, sync};`
best use `rake format`

Reply:
I just ran `cargo fmt`, but apparently `rake format` is slightly stricter in exactly this spot.